docs/en/setup/backend/backend-storage.md

```diff
@@ -211,6 +212,34 @@ All connection-related settings, including URL link, username, and password, are
 Only part of the settings is listed here. See the [HikariCP](https://github.com/brettwooldridge/HikariCP) connection pool document for full settings.
 To understand the function of the parameter `rewriteBatchedStatements=true` in MySQL, see the [MySQL official document](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-performance-extensions.html#cj-conn-prop_rewriteBatchedStatements).
 
+## MySQL-Sharding
```

Comment:
Is this only applicable to MySQL? Or is it applicable to other JDBC protocol?

Reply:
This PR only applicable to MySQL, postgre needs some adaptation and verification.

Reply:
We are not certain about ShardingSphere compatible capability crossing MySQL and PostgreSQL.
Kai has noticed and reported to their community, we could only work with ShardingSphere 5.1.2, but 5.2.0 doesn't work.